README.md

```diff
@@ -73,6 +73,18 @@ In some cases additional fields are provided:
 * `interfaces` is added, so configured IP addresses can be reused in icinga
 * `services` is added, so configured services can be reused in icinga
 
+### Tag split delimiter
+A Tag in netbox is only one value. But if your tags are imported by different hyperscaler, you have often a key value pair. So if you import this key value pair into netbox you have to chose a delimiter. In our case tags a are like:
+````
+thisisatag::true
+````
+<<<<<<< HEAD
```

Comment:
This doesn't seem right. Do you mind having another look at these conflict markers?

Reply:
Found it and resolved it